src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -324,17 +322,17 @@ void LedgerConsensusImp::mapCompleteInternal (
 }
 
 void LedgerConsensusImp::gotMap (
-    uint256 const& hash,
     std::shared_ptr<SHAMap> const& map)
 {
     std::lock_guard<std::recursive_mutex> _(lock_);
 
     try
     {
-        mapCompleteInternal (hash, map, true);
+        mapCompleteInternal (map, true);
     }
     catch (SHAMapMissingNode const& mn)
     {
+        // This should never happen
```

Comment:
If it shouldn't happen then we should consider restructuring the code. The problem with adding code that in theory should never happen, is that it becomes impossible to reach it via code coverage.